dashboard/src/components/AipReportsCard.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import { ref } from "vue";
+
+import { client, getPath } from "@/client";
+import { ResponseError } from "@/openapi-generator";
+import { useAipStore } from "@/stores/aip";
+import { useAuthStore } from "@/stores/auth";
+
+const aipStore = useAipStore();
+const authStore = useAuthStore();
+const reportKey = ref("");
+const error = ref("");
+
+const downloadDeletionReport = async () => {
+  if (!aipStore.current?.uuid) {
+    return;
+  }
```

Comment:
The UUID is required, I'd look at `aipStore.current?.deletionReportKey`, or remove the check entirely since we already check that to display the template.